extensions/cornerstone/src/init.js

```diff
@@ -102,6 +102,32 @@ export default function init({ servicesManager, configuration }) {
     csTools.BrushTool,
   ];
 
+  /* Measurement Service Events */
+  cornerstone.events.addEventListener(
+    cornerstone.EVENTS.ELEMENT_ENABLED,
+    event => {
+      MeasurementService.subscribe(
+        /* Should subscription event names be handled outside like here? Where to keep available subscription event names? */
+        csTools.EVENTS.MEASUREMENT_ADDED,
```

Comment:
This should probably be an event maintained by the MeasurementService?

```
MeasurementService.events.MEASUREMENT_CREATED
```

Reply:
Yeah, that makes sense to me as well.

Reply:
Updated.